app/experimenter/experiments/models.py

```diff
@@ -521,6 +521,13 @@ def monitoring_dashboard_url(self):
             to_date=end_date.strftime("%Y-%m-%d"),
         )
 
+    @property
+    def rollout_monitoring_dashboard_url(self):
+        if self.start_date and ((datetime.date.today() - self.start_date).days >= 1):
```

Comment:
Why do we need the `days >= 1` check?  And if we're checking for the presence of a `start_date`, couldn't we just check that `self.status == self.STATUS.LIVE`?